libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/CohortStatsHeatmap.tsx

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  defaultModelAssessmentContext,
+  ModelAssessmentContext,
+  HeatmapHighChart,
+  JointDataset,
+  generateMetrics,
+  ErrorCohort
+} from "@responsible-ai/core-ui";
+import { localization } from "@responsible-ai/localization";
+import { IDropdownOption } from "office-ui-fabric-react";
+import React from "react";
+
+import { generateCohortsStatsTable, wrapYAxisLabels } from "./StatsTableUtils";
+
+interface ICohortStatsHeatmapProps {
+  cohorts: ErrorCohort[];
+  selectableMetrics: IDropdownOption[];
+  selectedMetrics: string[];
+}
+
+class ICohortStatsHeatmapState {}
+
+export class CohortStatsHeatmap extends React.Component<
+  ICohortStatsHeatmapProps,
+  ICohortStatsHeatmapState
+> {
+  public static contextType = ModelAssessmentContext;
+  public context: React.ContextType<typeof ModelAssessmentContext> =
+    defaultModelAssessmentContext;
+
+  public render(): React.ReactNode {
+    const columns: string[] = [
+      localization.ModelAssessment.ModelOverview.countColumnHeader
+    ];
+    columns.push(
+      ...this.props.selectableMetrics
+        .filter((element) =>
+          this.props.selectedMetrics.includes(element.key.toString())
+        )
+        .map((element) => {
+          return element.text;
+        })
+    );
+
+    // generate table contents
+    const cohortLabeledStatistics = generateMetrics(

```

Comment:
I'm not excited about using `JointDataset` here but I don't see this elsewhere. Any idea @xuke444 ?